### PR TITLE
Allow invitation role to be updated via User Management page

### PR DESCRIPTION
**Pull request by acrellin**
invitations-1

invitations-3

invitations-4

Closes https://github.com/skyportal/skyportal/issues/1998

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]
LGTM!

**Comment by acrellin** [comment]
Need to update one API test here - will push in a sec

**Comment by acrellin** [comment]
Failure unrelated: 
```
=========================== short test summary info ============================
FAILED skyportal/tests/frontend/test_followup_requests.py::test_submit_new_followup_request_SEDM
= 1 failed, 435 passed, 1 skipped, 3 xfailed, 1 xpassed, 200 warnings, 2 rerun in 2182.10s (0:36:22) =
```

File: skyportal/tests/api/test_invitations.py [hunk 1]
```diff
@@ -120,6 +118,15 @@ def test_patch_invitation(
     )
     assert status == 200
 
+    # Try updaint role to View only
```
Comment (kmshin1397) [inline skyportal/tests/api/test_invitations.py:121]:
```suggestion
    # Try updating role to View only
```

**Review by kmshin1397 — COMMENTED** [review]
Small typo but otherwise looks good!

**Review by kmshin1397 — APPROVED** [review]
Feel free to self-merge once the typo is fixed

File: skyportal/tests/api/test_invitations.py (hunk 1 above)
Comment (acrellin) [reply skyportal/tests/api/test_invitations.py]: Thanks!

**Pull request merged**